content/en/users/fedcloud/solutions/ecas.md

```diff
@@ -0,0 +1,254 @@
+---
+title: "ECAS"
+type: docs
+weight: 20
+description: >
+  Using Elastic Cloud Computing Cluster (EC3) platform to create
+  ECAS environment.
+---
+
+The following guide is intended for researchers who want to use ECAS, a
+complete environment enabling data analysis experiments, in the EGI
+cloud.
+
+ECAS (ENES Climate Analytics Service) is part of the EOSC-hub service
+catalog and aims to:
+
+1.  provide server-based computation,
+2.  avoid data transfer, and
+3.  improve reusability of data and workflows (FAIR approach).
+
+It relies on [Ophidia](http://ophidia.cmcc.it/), a data analytics
+framework for eScience, which provides declarative, server-side, and
+parallel data analysis, jointly with an internal storage model able to
+efficiently deal with multidimensional data and a hierarchical data
+organization to manage large data volumes ("datacubes"), and on
+JupyterHub, to give users access to ready-to-use computational
+environments and resources.
+
+Thanks to the Elastic Cloud Compute Cluster (EC3) platform, operated by
+the [Polytechnic University of Valencia
+(UPV)](http://www.upv.es/index-en.html), researchers will be able to
+rely on the EGI Cloud Compute service to scale up to larger simulations
+without being worried about the complexity of the underlying
+infrastructure.
+
+This guide will show how to:
+
+-   deploy an ECAS elastic cluster of VMs in order to automatically
+    install and configure the whole ECAS environment services, i.e.
+    JupyterHub, PyOphidia, several Python libraries such as numpy,
+    matplotlib and Basemap;
+-   perform data intensive analysis using the Ophidia HPDA framework;
+-   access the ECAS JupyterHub interface to create and share documents
+    containing live code, equations, visualizations and explanatory
+    text.
+
+## Deploy an ECAS cluster with EC3
+
+In the latest release of the EC3 platform, tailored to support the EGI
+Applications on Demand (AoD) service, a new Ansible receipt is now
+available for researchers interested to deploy ECAS cluster on the EGI
+Infrastuctrure. Additional details on how to configure and deploy an
+ECAS cluster on EGI resources are provided in the next sections.
+
+ECAS in now available in the latest release of the EC3 platform
+supporting the EGI Applications on Demand (AoD). The next sections
+provide details on how to configure and deploy an ECAS cluster on EGI
+resources.
+
+### Configure and deploy the cluster
+
+To configure and deploy a Virtual Elastic Cluster using EC3aaS, access
```

Comment:
Looks like it's the first time EC3aaS is mentioned here without any introduction, it feels a bit weird